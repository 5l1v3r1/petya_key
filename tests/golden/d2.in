rkNK8Wp58MMCTFJ8PFVE4qucSs2fkkwKnKS5RJ9uiqpR9MqZ3ibEfHXefzAcD4CzLMh2qb5f8xMCNLHjdrr9WSwPudRJ636q
