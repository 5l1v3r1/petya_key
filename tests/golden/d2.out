8e7731b9c62332ce4b829380d3443804
